openupgrade_scripts/scripts/base/16.0.1.3/pre-migration.py

```diff
@@ -93,3 +99,19 @@ def migrate(cr, version):
             """,
         )
     openupgrade.rename_tables(cr, [("ir_translation", "_ir_translation")])
+    # Rename field's content of a model's name which has been changed
+    # Ex: parent_res_model of rating_rating table store model's name
+    for table, column in _RENAMED_CHANGED_MODELS_NAME:
+        if not openupgrade.table_exists(cr, table) or not openupgrade.column_exists(
+            cr, table, column
+        ):
+            continue
+        for old_model, new_model in changed_models.items():
+            openupgrade.logged_query(
+                cr,
+                f"""
+                UPDATE {table}
+                    SET {column} = REPLACE({column}, '{old_model}', '{new_model}')
+                WHERE {column} ilike '%{old_model}%'
```

Comment:
@duong77476 phải tìm là '=' chứ ilike sẽ sai trong tình huống nào đó
ví dụ `account.move` -> `viin.account.move` và `account.move.line` và `viin.account.move.line`
```suggestion
                WHERE {column} = '{old_model}'
```

Reply:
@royleviindoo Vâng a